### PR TITLE
Polymorphism

### PR DESCRIPTION
**Pull request by sooraj139**

### CODE REVIEW

File: opt-may-2016/Sooraj/com/training/polymorphism/DMV.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.training.polymorphism;
+
+public class DMV {
+	public void Truck1(Vehicle v){
```
Comment (saba83rish) [inline opt-may-2016/Sooraj/com/training/polymorphism/DMV.java]: why is there a Truck1 method in DMV? shouldnt this be register?

File: opt-may-2016/Sooraj/com/training/polymorphism/CarWash.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package com.training.polymorphism;
+
+public class CarWash {
+	public void wash1(Car c) {
```
Comment (saba83rish) [inline opt-may-2016/Sooraj/com/training/polymorphism/CarWash.java:4]: wash1 method is doing everything except wash..atleast print the washing process

Comment (sooraj139) [inline opt-may-2016/Sooraj/com/training/polymorphism/CarWash.java:4]: I have made all the changes.Please merge

File: opt-may-2016/Sooraj/com/training/polymorphism/ToyotaDealer.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.training.polymorphism;
+
+public class ToyotaDealer {
+	public void implement(Vehicle T){
```
Comment (saba83rish) [inline opt-may-2016/Sooraj/com/training/polymorphism/ToyotaDealer.java]: write a method name
variable name should be meaningful name

Comment (sooraj139) [inline opt-may-2016/Sooraj/com/training/polymorphism/ToyotaDealer.java]: Changes have been made.

**Pull request merged**